quint/test/types/inferrer.test.ts

```diff
@@ -395,6 +397,25 @@ module B {
     )
   })
 
+  // TODO
```

Comment:
Is this meant to be here? It's fine if you are leaving it for the next PR, I just want to make sure it is intentional

Reply:
Yep. This is I commented again ina followup.